ios/FluentUI/BottomSheet/BottomSheetViewController.swift

```diff
@@ -242,3 +243,18 @@ public class BottomSheetViewController: UIViewController {
         }
     }
 }
+
+extension BottomSheetViewController: UIGestureRecognizerDelegate {
+    public func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldRecognizeSimultaneouslyWith otherGestureRecognizer: UIGestureRecognizer) -> Bool {
+        let shouldRecognizeSimultaneously = false
```

Comment:
why even have this variable if it let and you are just going to return in line 258

Reply:
Will update this.

Reply:
i didn;t realise it was not using it, before raising the PR...my bad